meson.build

```
@@ -139,6 +139,11 @@ add_project_link_arguments(
   language: 'c'
 )
 
+# rust is optional
+if add_languages('rust', native: false, required: get_option('rust').enabled())
```

I think you got this wrong from how I expected.  See if this makes more sense:
```suggestion
rustc= dependency('rustc', required: get_option('rust'))
if add_languages('rust', native: false, required: rustc.found())
```